components/aws/sagemaker/unit_tests/run_all_tests.sh

```diff
@@ -0,0 +1,4 @@
+export PYTHONPATH=../
```

Comment:
This is set inside the Dockerfile, it seems unnecessary here.

Reply:
this way we can run this on local machine too